apps/web/vibes/soul/form/number-input/index.tsx

```diff
@@ -2,20 +2,22 @@
 
 import { clsx } from 'clsx';
 import { Minus, Plus } from 'lucide-react';
-import { forwardRef } from 'react';
+import * as React from 'react';
 
 import { FieldError } from '@/vibes/soul/form/field-error';
 import { Label } from '@/vibes/soul/form/label';
 
-export const NumberInput = forwardRef<
+export const NumberInput = React.forwardRef<
   React.ComponentRef<'input'>,
-  React.ComponentPropsWithoutRef<'input'> & {
+  Omit<React.ComponentPropsWithoutRef<'input'>, 'id'> & {
```

Comment:
Weren't you planning on renaming these sort of component to, e.g., `NumberInputField`? If we do that, then having the props be all the input props is probably something we don't want to do. As it stands right now, though, we can probably continue to take in `id` and if it's not passed then we can default to `useId`

Reply:
I will refactor this later.